Makefile

```diff
@@ -15,45 +15,40 @@ GO_DEPENDENCIES = golang.org/x/tools/cmd/goimports \
 				  golang.org/x/tools/cmd/stringer \
 				  github.com/mdempsky/go114-fuzz-build
 
-BREW_DEPENDENCIES = go-swagger@0.30.0 \
-					grype@0.40.1 \
-					cli@0.1.35 \
-					trivy@0.29.2
+SCRIPT_DEPENDENCIES = swagger \
+					protoc \
+					grype \
+					trivy \
+					ory
 
 define make-go-dependency
   # go install is responsible for not re-building when the code hasn't changed
-  tools/$2: .bin/gobin/go.mod .bin/gobin/go.sum Makefile
-		cd .bin/gobin; GOBIN=$(PWD)/.bin/gobin go install $1
+  .bin/$2: .bin/go.mod .bin/go.sum
+		cd .bin; GOBIN=$(PWD)/.bin go install $1
 endef
 $(foreach dep, $(GO_DEPENDENCIES), $(eval $(call make-go-dependency,$(dep),$(notdir $(dep)))))
 
-tools/yq: .bin/gobin/go.mod .bin/gobin/go.sum Makefile
-	cd .bin/gobin; GOBIN=$(PWD)/.bin/gobin go install github.com/mikefarah/yq/v4
-
-define make-brew-dependency
-  tools/$(firstword $(subst @, ,$(notdir $1))): tools/brew Makefile
-		HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=true HOMEBREW_NO_AUTO_UPDATE=1 brew install keto/tools/$1
+define make-script-dependency
+  # each script is responsible to figure out whether it should re-install
+  .PHONY: .bin/$1
+  .bin/$1:
+		./scripts/install-$1.sh
 endef
-$(foreach dep, $(BREW_DEPENDENCIES), $(eval $(call make-brew-dependency,$(dep))))
+$(foreach dep, $(SCRIPT_DEPENDENCIES), $(eval $(call make-script-dependency,$(dep))))
```

Comment:
The official install script would download new binaries regularly even if not needed, because the Makefile changes quite often. This might not be a problem when using a good network connection, but I was already annoyed multiple times when working on the go by this... That's why I build version checks into all scripts, and wrapped the official install scripts.